CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 This document outlines major changes between releases.
 
+## 0.114.0 "Chemosorption" (17 Nov 2025)
+
+This version fixes the state difference at block 11074661 of testnet caused by
+improper handling of native NEP-17 token transfer. It also fixes a deadlock
+caused by inability to terminate an expired iterator session. It's fully
+compatible with the C# node 3.8+ and delivers a set of improvements for
+compiler, NeoFS block storage audit and uploader along with a preview of Faun
+changes.
+
+Deprecated functionality has been removed according to the schedule (`math.Max`
+and `math.Min` interop functions along with `SessionExpirationTime` RPC server
+config), consider upgrading affected smart contracts and nodes config
+correspondingly. The default ports of Prometheus and pprof services have been
+changed for mainnet config. Testnet node operators need to resynchronize their
+DB (or reset the node's state to block 11074660 for full nodes).
```

Comment:
Maybe mention that state is the same at higher block and the only real difference now is two application logs for two transactions.